perf/src/perf_libs.rs

```diff
@@ -140,11 +140,13 @@ fn find_cuda_home(perf_libs_path: &Path) -> Option<PathBuf> {
     None
 }
 
-pub fn append_to_ld_library_path(path: String) {
-    let ld_library_path =
-        path + ":" + &env::var("LD_LIBRARY_PATH").unwrap_or_else(|_| "".to_string());
-    info!("setting ld_library_path to: {:?}", ld_library_path);
-    env::set_var("LD_LIBRARY_PATH", ld_library_path);
+pub fn append_to_ld_library_path(mut ld_library_path: String) {
+    if let Ok(env_value) = env::var("LD_LIBRARY_PATH") {
+        ld_library_path.push(':');
+        ld_library_path.push_str(&env_value);
+        info!("setting ld_library_path to: {:?}", ld_library_path);
+        env::set_var("LD_LIBRARY_PATH", ld_library_path);
+    }
```

Comment:
this isn't the same. If `LD_LIBRARY_PATH` starts off empty, we will not add to it here.

Reply:
nice catch! will fix momentarily

Reply:
fixed in [f3aba37](https://github.com/solana-labs/solana/pull/33000/commits/f3aba3707b9de7674cdbcfd8a7d94802f1fff7e7)